api/app/jobs/common_model_fetchers.py

```diff
@@ -453,10 +454,14 @@ def process(self, model_type: ModelEnum):
         # Get model runs that are complete (fully downloaded), but not yet interpolated.
         query = get_prediction_model_run_timestamp_records(
             self.session, complete=True, interpolated=False, model_type=model_type)
+        refresh_mat_view = False
         for model_run, model in query:
+            refresh_mat_view = True
```

Comment:
Minor nit, and not a strongly held opinion, but should we rename `refresh_mat_view` something like `model_processed` since that's what's driving whether we should refresh the mat view and also what's happening if we make it into the for block?

Reply:
Done! 
https://github.com/bcgov/wps/pull/3686/commits/dfce44b0184bc1bf87f39f82b8cd04c9524d5c00